packages/node_modules/@webex/redux-module-conversation/src/__snapshots__/actions.test.js.snap

```diff
@@ -513,10 +513,9 @@ exports[`redux-module-conversation actions #submitAdaptiveCardActions can start
 Array [
   Object {
     "payload": Object {
-      "activity": Object {
-        "activity": Object {
-          "id": "7erfber34",
-        },
+      "actionSubmitAcknowledgeObject": Object {
+        "cardActionAcknowledgmentId": undefined,
```

Comment:
This should have been set to `7erfber34` which you are defining in the mock spark call. Could you investigate why?